arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md

```diff
@@ -16,6 +16,14 @@ import PublicPreviewBannerPartial from './partials/_orbit-public-preview-banner-
 
 <PublicPreviewBannerPartial />
 
+### Arbitrum Orbit is Mainnet Ready:
+
+The Arbitrum Orbit core technology has undergone a comprehensive audit and is prepared for deployment on Mainnet. To optimize the experience, we recommend deploying an Orbit chain to Testnet before proceeding to Mainnet. 
```

Comment:
I think we want a stronger statement recommending that people should test before launching on mainnet 😁